upscollector.go

```diff
@@ -41,105 +44,119 @@ var _ prometheus.Collector = &UPSCollector{}
 	)
 
 	return &UPSCollector{
 		UPSLoadPercent: prometheus.NewDesc(
 			prometheus.BuildFQName(namespace, "", "ups_load_percent"),
 			"Current UPS load percentage.",
-			labels,
+			nil,
 			nil,
 		),
 
 		BatteryChargePercent: prometheus.NewDesc(
 			prometheus.BuildFQName(namespace, "", "battery_charge_percent"),
 			"Current UPS battery charge percentage.",
-			labels,
+			nil,
 			nil,
 		),
 
 		LineVolts: prometheus.NewDesc(
 			prometheus.BuildFQName(namespace, "", "line_volts"),
 			"Current AC input line voltage.",
-			labels,
+			nil,
 			nil,
 		),
 
 		LineNominalVolts: prometheus.NewDesc(
 			prometheus.BuildFQName(namespace, "", "line_nominal_volts"),
 			"Nominal AC input line voltage.",
-			labels,
+			nil,
 			nil,
 		),
 
 		BatteryVolts: prometheus.NewDesc(
 			prometheus.BuildFQName(namespace, "", "battery_volts"),
 			"Current UPS battery voltage.",
-			labels,
+			nil,
 			nil,
 		),
 
 		BatteryNominalVolts: prometheus.NewDesc(
 			prometheus.BuildFQName(namespace, "", "battery_nominal_volts"),
 			"Nominal UPS battery voltage.",
-			labels,
+			nil,
 			nil,
 		),
 
 		BatteryNumberTransfersTotal: prometheus.NewDesc(
 			prometheus.BuildFQName(namespace, "", "battery_number_transfers_total"),
 			"Total number of transfers to UPS battery power.",
-			labels,
+			nil,
 			nil,
 		),
 
 		BatteryTimeLeftSeconds: prometheus.NewDesc(
 			prometheus.BuildFQName(namespace, "", "battery_time_left_seconds"),
 			"Number of seconds remaining of UPS battery power.",
-			labels,
+			nil,
 			nil,
 		),
 
 		BatteryTimeOnSeconds: prometheus.NewDesc(
 			prometheus.BuildFQName(namespace, "", "battery_time_on_seconds"),
 			"Number of seconds the UPS has been providing battery power due to an AC input line outage.",
-			labels,
+			nil,
 			nil,
 		),
 
 		BatteryCumulativeTimeOnSecondsTotal: prometheus.NewDesc(
 			prometheus.BuildFQName(namespace, "", "battery_cumulative_time_on_seconds_total"),
 			"Total number of seconds the UPS has provided battery power due to AC input line outages.",
-			labels,
+			nil,
 			nil,
 		),
 
 		LastTransferOnBattery: prometheus.NewDesc(
 			prometheus.BuildFQName(namespace, "", "apcupsd_last_transfer_on_battery"),
 			"Time of last transfer to battery since apcupsd startup.",
-			labels,
+			nil,
 			nil,
 		),
 
 		LastTransferOffBattery: prometheus.NewDesc(
 			prometheus.BuildFQName(namespace, "", "apcupsd_last_transfer_off_battery"),
 			"Time of last transfer from battery since apcupsd startup.",
-			labels,
+			nil,
 			nil,
 		),
 
 		LastSelftest: prometheus.NewDesc(
 			prometheus.BuildFQName(namespace, "", "apcupsd_last_selftest"),
 			"Time of last selftest since apcupsd startup.",
-			labels,
+			nil,
 			nil,
 		),
 
 		NominalPowerWatts: prometheus.NewDesc(
 			prometheus.BuildFQName(namespace, "", "apcupsd_nominal_power_watts"),
 			"Nominal power output in watts.",
-			labels,
+			nil,
+			nil,
+		),
+
+		UPSStatus: prometheus.NewDesc(
```

Comment:
I feel like it would make more sense to include the status label in UPSInfo rather than in its own metric.